test/OpenTelemetry.Exporter.Jaeger.Tests/OpenTelemetry.Exporter.Jaeger.Tests.csproj

```diff
@@ -18,6 +18,7 @@
     <DotNetCliToolReference Include="dotnet-xunit" Version="$(DotNetXUnitCliVer)" />
     <PackageReference Include="Microsoft.Extensions.Hosting" Version="$(MicrosoftExtensionsHostingPkgVer)" />
     <PackageReference Include="Microsoft.Extensions.Http" Version="3.1.20" />
+    <PackageReference Include="RichardSzalay.MockHttp" Version="6.0.0" />
```

Comment:
we try to avoid adding extra packages, even in test projects. do we really this one?

Reply:
It is the easiest way I know to test an http client - I see the zipkin exporter tests use a `TestHttpServer`. I'll switch this test to use it.

Reply:
Thanks.

Reply:
No problem at all